### PR TITLE
Export similar.tsv from gxd.sqlite. create new gxd.sqlite from similar.tsv

### PR DESCRIPTION
**Pull request by anjakefala**
This PR is Ready for Review. It's a draft, because before we merge this PR, gxd repo needs to merge the new similar.tsv.

### CODE REVIEW

**Comment by anjakefala** [comment]
It takes 23.67 seconds to create the sqlite and 44 seconds to analyse 1000 puzzles. 

These days gxd seems to add about 6 files, so analysis would take ~24-30 seconds in additional runtime each day. I propose adding it to the daily pipeline.

File: Makefile [hunk 1]
```diff
@@ -72,10 +75,12 @@ commit:
 	git push)
 
 gridmatches: gxd.sqlite src/gridcmp.so
-	cat src/findmatches.sql | time sqlite3 gxd.sqlite
+	time python3 src/findmatches.py -N 100
```
Comment (saulpw) [inline Makefile:78]: This PR is missing src/findmatches.py

Comment (anjakefala) [reply Makefile:78]: Nice catch! I thought I committed in the previous PR, but I guess that explains why this one is so small.

**Review by saulpw — APPROVED** [review]

File: scripts/27-pubyear-stats.py [hunk 2]
```diff
@@ -138,19 +138,22 @@ def process_counter(count, comp_value):
...
                 pct = int(r.match_pct)
-                if dt2 < dt1:  # only capture the later one
+                # only check for similarity for crosswords that have recorded solutions
+                xd1_A1 = all(c == 'X' for c in xd1.get_answer("A1"))
+                xd1_D1 = all(c == 'X' for c in xd1.get_answer("D1"))
+                if dt2 < dt1 and not (xd1_A1 and xd1_D1):  # only capture the later one
                     ##deduce_similarity_type
                     if diff_authors(aut1, aut2): # suspicious
-                        if pct >= 50:
+                        if abs(pct) >= 50:
```
Comment (saulpw) [inline scripts/27-pubyear-stats.py:147]: How about `pct = abs(pct)` above this block?

**Review by saulpw — APPROVED** [review]
Thanks so much for putting this together, @anjakefala !

**Pull request merged**